pkg/login/ldap.go

```diff
@@ -54,6 +54,17 @@ func (a *ldapAuther) login(query *LoginUserQuery) error {
 	if ldapUser, err := a.searchForUser(query.Username); err != nil {
 		return err
 	} else {
+
+		for _, ldapGroup := range a.server.LdapGroups {
```

Comment:
I generally do not like to add more options to a config, but you are right there should be an option to switch between different LDAP schemes. 
Something like `search_in_groups = true/false`. What do you think, is this name understandable?